source/change-log/index.html.md.erb

```diff
@@ -7,6 +7,21 @@ description: Find out what has changed in the specification and make sure your a
 
 # Change log
 
+### Version 3.2
+
+#### 1 August 2023
+
+ * For the mobile application via server and mobile application direct connection methods, you do not need to provide a
+   value for `os` in Gov-Client-User-IDs. If your application submits values for `os` it is not an issue.
```

Comment:
I don't think we need new lines here. Other changes have it continuous one line

Reply:
Do you not think it's more readable? It doesn't affect the rendering.